standup/auth0/models.py

```diff
@@ -3,11 +3,11 @@
 
 
 class IdToken(models.Model):
-    """Relates an ID token to a user
+    """Associates a user with their id token
 
-    These tokens need to be renewed on a periodic basis and if they fail, they cause the Django
-    session to expire.
+    These tokens need to be renewed on a periodic basis and if they fail, they force Django logout.
 
     """
     user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
-    id_token = models.TextField(max_length=100, null=True)
+    id_token = models.TextField(help_text='The id token for this user.')
```

Comment:
Mmm... I don't think we ever do that. As the code stands now, we query that table here and that's it:

* https://github.com/mozilla/standup/pull/325/files#diff-a1cf37ebb60cbdd2c99767226e989f5dR90
* https://github.com/mozilla/standup/pull/325/files#diff-6776186224a7b049a218c30b0cf2f46eR196

The only place where we query by `id_token` is in the tests.

So I think this is ok as it stands now, but I'm glad you mentioned it and we double-checked.